File: src/util/fibonacci.c
#include <stdlib.h>
#include <stdio.h>
#include "emscripten/emscripten.h"

int main(void) {
    printf("WebAssembly Main Loaded.");
};

EMSCRIPTEN_KEEPALIVE
int createFibonacci(int n) {
  if (n == 1) {
      return 1;
  } else if(n == 2) {
      return 1;
  } else {
      return createFibonacci(n-1) + createFibonacci(n-2);
  }
}

File: src/util/fibonacci.js
export function createFibonacci(n) {
    if (n === 1) {
        return 1;
    } else if(n === 2) {
        return 1;
    } else {
        return createFibonacci(n-1) + createFibonacci(n-2);
    }
  }